src/appBootstrapper.jsx

```diff
@@ -50,6 +50,10 @@ initializeConfiguration();
 // THIS TELLS WEBPACK BUNDLE LOADER WHERE TO LOAD SPLIT BUNDLES
 __webpack_public_path__ = AppConfig.frontendUrl;
 
+if (localStorage.heroku && localStorage.localdev !== "true") {
+    __webpack_public_path__ = ['//',localStorage.heroku,'.herokuapp.com','/'].join('');
+}
```

Comment:
this prolly shouldn't be in this PR

Reply:
will do a rebase and remove those unrelated. sorry!